packages/jest-bolt-receiver/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack-wrench/jest-bolt-receiver",
-  "version": "2.0.0",
+  "version": "1.1.1",
```

Comment:
The upgrade path is different now that the packages are at different levels. 

We can't use lerna to do the upgrades unless we switch to 100% conventional commits. Which we probably should. `2.0.0` is already published.